components/scream/cime_config/testdefs/testmods_dirs/scream/pio2/shell_commands

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+./xmlchange PIO_VERSION=2
```

Comment:
Isn't this the default for everything now? Is this line needed after our recent upstream merge?

Reply:
Oh yes, good point.  We can get rid of both this and the "testmods" comment above.  I'll push the change.